.trivyignore

```diff
@@ -1,9 +1,25 @@
 # telegraf vulnerabilities
 CVE-2019-3826
-CVE-2022-29190
-CVE-2022-29222
-CVE-2022-29189
 CVE-2022-1996
 
-#dpkg vulnerability in ubuntu
+# still present in mdsd telegraf
+CVE-2021-42836
```

Comment:
@pfrcks , these vulns not required here for dev since we have fixed all these, right?